membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -61,54 +69,78 @@ class AttributeController(attributesFromZuora: AttributesFromZuora, commonAction
     else Future.successful(None)
   }
 
+  def getLatestMobileSubscription(identityId: String)(implicit executionContext: ExecutionContext): Future[Option[MobileSubscriptionStatus]] = {
+    mobileSubscriptionService.getSubscriptionStatusForUser(identityId).transform {
+      case Failure(NonFatal(error)) =>
+        log.warn("Exception while fetching mobile subscription, assuming none", error)
+        Success(None)
+      case Success(-\/(error)) =>
+        log.warn(s"Unable to fetch mobile subscription, assuming none: $error")
+        Success(None)
+      case Success(\/-(status)) => Success(status)
+    }
+  }
+
+  def enrichZuoraAttributes(zuoraAttributes: Option[Attributes], latestOneOffDate: Option[LocalDate], mobileSubscriptionStatus: Option[MobileSubscriptionStatus]): Option[Attributes] = {
```

Comment:
enrich doesn't really mean anything specific, could we call it withNonZuoraAttributes or something similar

Reply:
"Enrich" doesn't mean enough if you stop reading at the method name, but if you read the method + the parameters then I'm of the opinion it carries the intended meaning.

How strongly do you feel about this one?

Reply:
if you've thought about it, it's enough for me!  😆 So no need to change it.